### PR TITLE
Upgrade tests

### PR DESCRIPTION
**Pull request by SilkovAlexander**

### CODE REVIEW

File: .ci/Jenkinsfile.gosh-prepare-rc [hunk 1]
```diff
@@ -158,7 +158,7 @@ pipeline {
                         } else {
                             sh """      
                                 aws s3api create-bucket --bucket ${BUCKET_NAME} --region eu-west-2 --acl public-read --create-bucket-configuration LocationConstraint=eu-west-2
-                                aws s3 website s3://${BUCKET_NAME} --index-document index.html --error-document error.html
+                                aws s3 website s3://${BUCKET_NAME} --index-document index.html --error-document index.html
```
Comment (andrew-kurochkin) [inline .ci/Jenkinsfile.gosh-prepare-rc:161]: Why do we replace error document here?

File: git-remote-gosh/src/blockchain/mod.rs [hunk 2]
```diff
@@ -94,6 +113,13 @@ struct CallResult {
     out_msgs: Vec<String>,
 }
 
+#[derive(Deserialize, Debug)]
+struct FFCallResult {
```
Comment (andrew-kurochkin) [inline git-remote-gosh/src/blockchain/mod.rs:117]: Please rename according to the method name

**Review by andrew-kurochkin — CHANGES_REQUESTED** [review]

File: .ci/Jenkinsfile.gosh-prepare-rc (hunk 1 above)
Comment (SilkovAlexander) [reply .ci/Jenkinsfile.gosh-prepare-rc:161]: This change appeared after merging dev to this branch

File: git-remote-gosh/src/blockchain/mod.rs (hunk 2 above)
Comment (SilkovAlexander) [reply git-remote-gosh/src/blockchain/mod.rs:117]: ^

Comment (SilkovAlexander) [reply git-remote-gosh/src/blockchain/mod.rs:117]: Also from dev blame says it was Vasiliy

**Pull request merged**